coffeecard/CoffeeCard.Models/Entities/MenuItem.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+
+namespace CoffeeCard.Models.Entities;
+
+/// <summary>
+/// An item or good that Analog offers, such as a cappuccino or a t-shirt.
+/// </summary>
+public class MenuItem
```

Comment:
It should have a uniqueness index to avoid having items with the same name

```suggestion
[Index(nameof(Name), IsUnique = true)]
public class MenuItem
```

Reply:
Addressed in fe39ffed2d7d416aee95f353586cf80e70afeb6f